### ISSUE TITLE
Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by lexicalunit** [AtomLinter/linter-coffeelint#93]
This occurs when it tries to lint a `.cson` file. I'm using Atom verson `1.20.0-beta1`.

Copy/pasted from the developer tools command line:

```
events.js:106 Cannot convert undefined or null to object TypeError: Cannot convert undefined or null to object
  at Function.keys (<anonymous>:null:null)
  at configImportsModules (/Users/me/.atom/packages/linter-coffeelint/src/worker.js:25:10)
  at Object.module.exports (/Users/me/.atom/packages/linter-coffeelint/src/worker.js:67:9)
  at process.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app/src/task-bootstrap.js:51:41)
  at emitTwo (events.js:106:13)
  at process.emit (events.js:191:7)
  at process.nextTick (internal/child_process.js:752:12)
  at _combinedTickCallback (internal/process/next_tick.js:67:7)
  at process._tickCallback (internal/process/next_tick.js:98:9)
- (anonymous) @ <embedded>:71331
- (anonymous) @ <embedded>:71405
- module.exports.Emitter.simpleDispatch @ <embedded>:63624
- module.exports.Emitter.emit @ <embedded>:63740
- (anonymous) @ <embedded>:71357
- emitTwo @ events.js:106
- emit @ events.js:191
- process.nextTick @ internal/child_process.js:752
- _combinedTickCallback @ internal/process/next_tick.js:67
- _tickCallback @ internal/process/next_tick.js:98

linter-registry.js [sm]:137 [Linter] Error running CoffeeLint Cannot convert undefined or null to object
- (anonymous) @ linter-registry.js [sm]:137
- _tickCallback @ internal/process/next_tick.js:103

events.js:106 Cannot convert undefined or null to object TypeError: Cannot convert undefined or null to object
  at Function.keys (<anonymous>:null:null)
  at configImportsModules (/Users/me/.atom/packages/linter-coffeelint/src/worker.js:25:10)
  at Object.module.exports (/Users/me/.atom/packages/linter-coffeelint/src/worker.js:67:9)
  at process.<anonymous> (/Applications/Atom Beta.app/Contents/Resources/app/src/task-bootstrap.js:51:41)
  at emitTwo (events.js:106:13)
  at process.emit (events.js:191:7)
  at process.nextTick (internal/child_process.js:752:12)
  at _combinedTickCallback (internal/process/next_tick.js:67:7)
  at process._tickCallback (internal/process/next_tick.js:98:9)
- (anonymous) @ <embedded>:71331
- (anonymous) @ <embedded>:71405
- module.exports.Emitter.simpleDispatch @ <embedded>:63624
- module.exports.Emitter.emit @ <embedded>:63740
- (anonymous) @ <embedded>:71357
- emitTwo @ events.js:106
- emit @ events.js:191
- process.nextTick @ internal/child_process.js:752
- _combinedTickCallback @ internal/process/next_tick.js:67
- _tickCallback @ internal/process/next_tick.js:98

linter-registry.js [sm]:137 [Linter] Error running CoffeeLint Cannot convert undefined or null to object
- (anonymous) @ linter-registry.js [sm]:137
- _tickCallback @ internal/process/next_tick.js:103
```

**Comment by Arcanemagus** [comment]
What version of CoffeeLint is in that project?

**Comment by lexicalunit** [comment]
I've got it installed globally at `/usr/local/lib/node_modules/coffeelint/bin/coffeelint`. Version is `1.16.0`.

**Comment by Arcanemagus** [comment]
Should be fixed in v1.3.1, thanks for the report!
